src/ductless.ts

```diff
@@ -306,12 +333,40 @@ export class KumoPlatformAccessory_ductless {
         return false;
       }
       this.platform.log.debug('%s (queryDevice_Direct): success.', this.accessory.displayName);
+
+      if (this.Humdity) {
+        this.platform.log.debug('querying external sensors on %s', this.accessory.context.serial);
+        const sensorData = await this.platform.kumo.queryDeviceSensors_Direct(this.accessory.context.serial);
+        if (sensorData) {
+          this.accessory.context.sensors = sensorData;
+        }
+      }
     }
 
     // update device contect
     this.accessory.context.device = device;          
     return true;
-  } 
+  }
+
+  private updateCurrentRelativeHumidity() {
+    if (!this.Humdity) {
+      return;
+    }
+    let currentValue: number = <number>this.Humdity.getCharacteristic(this.platform.Characteristic.CurrentRelativeHumidity).value;
+    if (this.accessory.context.sensors && this.accessory.context.sensors.length) {
+      const ourSensor = this.accessory.context.sensors[0];
+      currentValue = ourSensor.humidity;
+      this.platform.log.debug('setting humidity to %s', currentValue);
+
+      if (ourSensor.battery) {
+        this.Humdity.updateCharacteristic(this.platform.Characteristic.StatusLowBattery, ourSensor.battery < 10);
```

Comment:
Would it be possible to add a battery level characteristic as well, in addition to the this.platform.Characteristic.StatusLowBattery characteristic? I believe there's one in the spec

Reply:
I looked for one, but I only saw a low battery alert: https://developers.homebridge.io/#/service/HumiditySensor

Reply:
I think it might be this? https://developers.homebridge.io/#/service/Battery

Reply:
Ahhh good call. I'll experiment with that tonight and see what I can make.

Reply:
Got this working...added a battery service and it seems to work as expected.

![image](https://user-images.githubusercontent.com/1617612/176809810-641618c5-265e-4764-b34b-490d475597e0.png)
![image](https://user-images.githubusercontent.com/1617612/176809841-88d32bfc-255e-4c64-91b6-c733719b58e9.png)